### PR TITLE
[RPM/2.3] - Add nodejs-redhat-cloud-services-frontend-components package

### PR DESCRIPTION
**Pull request by laviro**

### CODE REVIEW

**Comment by ShimShtein** [comment]
[test]

**Review by evgeni — CHANGES_REQUESTED** [review]
I've requested changes in #6158 -- will need to be ported here too.

**Comment by laviro** [comment]
Thanks @evgeni !! updated also here :)

File: rel-eng/tito.props [hunk 1]
```diff
@@ -1138,4 +1140,4 @@ whitelist = katello
   rubygem-runcible
   rubygem-stomp
   rubygem-typhoeus
-  rubygem-zest
+  rubygem-zest
```
Comment (evgeni) [inline rel-eng/tito.props:1143]: can you clear out that whitespace? :)

**Comment by ShimShtein** [comment]
@evgeni, done.

**Review by evgeni — APPROVED** [review]

**Pull request merged**